scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -429,7 +429,12 @@ s32 BPF_STRUCT_OPS(layered_select_cpu, struct task_struct *p, s32 prev_cpu, u64
 	goto out_put_idle_smtmask;
 
 dispatch_local:
-	tctx->dispatch_local = true;
+	lstat_inc(LSTAT_LOCAL, layer, cctx);
+	/* If the target is this CPU, just dispatch right here to avoid enqueue() */
+	if (cpu == bpf_get_smp_processor_id())
```

Comment:
Oh, if you dispatch to `SCX_DSQ_LOCAL` from `.select_cpu()`, it will dispatch to the local DSQ of the CPU returned by `.select_cpu()`, so it can always dispatch directly and drop `.dispatch_local` field.

Reply:
Ah, I misunderstood that, will fix